catalog-info.yaml

```diff
@@ -0,0 +1,21 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'openedx-calc'
+  description: >
+          A helper library for mathematical calculations and
+          symbolic mathematics, used by the edx-platform.
+  links:
+    - url: "https://github.com/openedx/openedx-calc"
+      title: "openedx/openedx-calc"
+      icon: "GitHub"
+  annotations:
+    openedx.org/arch-interest-groups: ""
+    openedx.org/release: "main"
```

Comment:
@ichintanjoshi you need to look at the existing `openedx.yaml` to determine whether or not a repo needs this annotation.  This repo was not getting tagged previously so it should not get tagged now. 
```suggestion
```